src/components/tasks/TaskList/TaskList.tsx

```diff
@@ -0,0 +1,57 @@
+import beautifyTaskStatus from "@/helperFunctions/beautifyTaskStatus";
+import task from "@/interfaces/task.type";
+import { useState } from "react";
+import Card from "../card";
+import styles from "../card/card.module.scss";
```

Comment:
can we sort the imports here somewhat like in the format where
the library imports are first
then components
then CSS followed by assets

Reply:
Imports added in order
```jsx
Library
Components
Assets
CSS
```